src/Resolution.test.ts

```diff
@@ -1,11 +1,14 @@
 import nock from 'nock';
 import Resolution, { ResolutionErrorCode } from './index';
 import { UnclaimedDomainResponse, NamingServiceName } from './types';
+var Web3WsProvider = require('web3-providers-ws');
+var Web3HttpProvider = require('web3-providers-http');
```

Comment:
Why `var` and `require` though?

Reply:
I tried to make it with an import instead of require and typescript couldn't understand the Web3HttpProvider, I tried to import HttpProvider as in class name and got errors that it doesn't have a constructor =-( So I reverted to require

Reply:
Maybe `import Web3HttpProvider = require('web3-providers-http');` will work?

Reply:
Seems like they export them in CommonJS way, so it should work

Reply:
This is what i got whenever i change to import. 
I tried with 
`Maybe import Web3HttpProvider = require('web3-providers-http');`
Same issue
<img width="1144" alt="Screen Shot 2020-06-15 at 11 57 46 AM" src="https://user-images.githubusercontent.com/16522024/84695280-71f50d00-aeff-11ea-95a9-6fe633c0067d.png">